towhee/engine/factory.py

```diff
@@ -101,6 +101,10 @@ def get_op(self):
         self.__check_init__()
         return self._op
 
+    def get_op_config(self):
```

Comment:
please use `@property` instead of `get_xxx`

Reply:
fixed